### DISCUSSION TITLE
[v3] How to use AlpineJS separately?

### DISCUSSION THREAD

**Discussion by CorvS** [livewire/livewire#6818] [1. Help] [answered]
Hello!

Livewire 3 ships with AlpineJS by default.
How am I supposed to use AlpineJS separately on pages where I don't need the full power of Livewire?

If you take Laravel Breeze for example. 
The `<x-layouts.navigation />` component stops working correctly on pages without Livewire components.

Of course I could simply replace it with a Livewire `<livewire:navigation />` component, but that seems a bit dirty to me.

Would love to hear your thoughts.

**Comment by joshhanley** [comment, marked as answer]
@CorvS you could just manually include Livewire scripts in your app layout, which will mean Livewire and Alpine are accessible on all pages. Otherwise you will need a second layout that includes Alpine that you use for pages that don't have Livewire.

Hope this helps!

**Reply by CorvS** [reply]
@joshhanley OK, thanks for the quick reply!
I was hoping I missed a part in the docs that talks about this specific use case and that I can avoid manually adding the scripts to the app layout, but I guess I'm just nitpicking here.
